### ISSUE TITLE
Constant condition is not detected when used with `and` or `or`

### ISSUE THREAD

**Issue by sobolevn** [wemake-services/wemake-python-styleguide#1880]
This code does not raise:

```python
if some or True: ...
```

This one raises:

```python
if True: ...
```

```
  1:4      WPS314 Found conditional that always evaluates the same
  if True: ...
```
